### ISSUE TITLE
Warlock: Affliction: Typo in Impending Catastrophe condition

### ISSUE THREAD

**Issue by tflo** [Hekili/hekili#1213]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

I haven’t played Affliction for a while, so not sure if this is the same typo you were referring in https://github.com/Hekili/hekili/issues/1103#issuecomment-849547713, but please have a look at the Impending Catastrophe condition (the one w/o Phantom Singularity):

```
! talent.phantom_singularity.enabled & ( cooldown.summon_darkglare.remains < 10 | cooldown.summon_darkglare.remains > 50 & cooldown.summon_darkglare.remains > 25 & conduit.corrupting_leer.enabled )
```

For me, the “&” after “> 50” doesn’t make sense. I changed it to “|” and now Impending Catastrophe gets correctly recommended. (As it is above, it is never recommended if you don’t have the conduit enabled.)

Hekili v9.1.0-1.0.18-beta4.

– Tom

### How to Reproduce

We’re speaking about a probable typo here, so nothing to reproduce in first place.

To verify that Impending Catastrophe gets not recommended in-game, just do the regular Affliction rotation and see that it never gets recommended. Make sure you have the Phantom Singularity talent deselected and the Corrupting Leer conduit not enabled.

### Snapshot (Link)

https://pastebin.com/Ap31NWvA

**Comment by Hekili** [comment]
It's consistent with sim, but does look erroneous.

From sim profile:
`
actions.covenant=impending_catastrophe,if=!talent.phantom_singularity&(cooldown.summon_darkglare.remains<10|cooldown.summon_darkglare.remains>50&cooldown.summon_darkglare.remains>25&conduit.corrupting_leer)`

**Comment by Hekili** [comment]
Just want to confirm, I'll make the change so that the & after 50 becomes a |, but you should consider reporting this to the Warlock theorycrafters as well since the default simc profile should get updated, too.

**Comment by tflo** [comment]
> It's consistent with sim, but does look erroneous.

OK, that explains why Venthyr sims so bad ;)

**Comment by Hekili** [comment]
This fix is in 1.0.18.

**Comment by tflo** [comment]
Thank you.
